apps/src/code-studio/pd/application_dashboard/constants.js

```diff
@@ -87,3 +87,14 @@ export const ApplicationTypes = {
   teacher: 'Teacher',
   facilitator: 'Facilitator'
 };
+
+/**
+ * Application statuses for which we require a scholarship status
+ */
+export const ScholarshipStatusRequiredStatuses = [
```

Comment:
Is this Application status or Scholarship status? If it's Scholarship status then value like 'registration_sent' is a bit confusing.

Reply:
Oh I see, it's application status that requires scholarship status to be set.